### DISCUSSION TITLE
Welcome to newifi3-d2-openwrt Discussions!

### DISCUSSION THREAD

**Discussion by Jeffery186** [Jeffery186/newifi3-d2-openwrt#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 1453851506** [comment]
大佬新路由3，想用手机usb共享网络，配置usb接口运行不了显示设备出错，麻烦你能编译带有usb网卡驱动吗，麻烦你了
